README.md

```diff
@@ -95,12 +97,96 @@ variables within the `settings.yml` file as described in the Installation sectio
 
 - Store any Jinja2 variable definitions you wish to use in your configuration
   template files in `resources/settings.yml`.
+- Store any appcli stack specific variables in `resources/stack-settings.yml`.
 - Store your `docker-compose.yml`/`docker-compose.yml.j2` file in `resources/templates/baseline/`.
 - Configuration files (Jinja2 compatible templates or otherwise) can be stored in one
   of two locations:
   - `resources/templates/baseline` - for templates which the end user **is not** expected to modify.
   - `resources/templates/configurable` - for templates which the end user is expected to modify.
 
+### Backup and restore application configuration and data
+Backup and restore functionality can be configured in a `backup` block in`stack-settings.yml` 
+The available variables for the backup block is:
+| variable      | Description                                                                                                |
+| ------------- | ---------------------------------------------------------------------------------------------------------- |
+| numberOfBackupsToKeep | The number of local backups to keep. A rolling deletion strategy is used to remove the oldest backups once this number has been reached. Set this value to `0` to never delete a backup.                         |
+| ignoreList            | The ignore list is a list of glob patterns used to specify what files to exclude from the backup.  |
+| remote                | Defines all remote backups.                                                                        |
+
+
+    # filename: stack-settings.yml
+
+    backup:
+        numberOfBackupsToKeep: 0 
+        ignoreList: 
+        remote:
+
+#### Ignore lists
+The `ignoreList` list in `stack-settings.yml` can contain a list of glob patterns, if any of these patterns match a files full path it will be excluded from the backup. The following example will exclude any file in a directory that ends with `metastore`.
```

Comment:
Markdown renderers can be fussy with whitespace. So please include a blank line between:
- headers and text
```
### Ignore lists

The `ignoreList`....
```

- paragraphs and lists
``
The following are...

- Item 1
- Item 2

More text....
```